bucket/qtpass.json

```
@@ -0,0 +1,23 @@
+{
+    "version": "1.3.2",
+    "license": "GPL-3.0-only",
+    "homepage": "https://qtpass.org/",
+    "description": "GUI for pass, the standard UNIX password manager.",
+    "url": "https://github.com/IJHack/QtPass/releases/download/v1.3.2/qtpass-1.3.2.exe",
+    "hash": "0fb9e68ce7e52e614ebb29dd9fc0b06b24f4262e9509090853e2c2991c171e72",
+    "innosetup": true,
+    "bin": "qtpass.exe",
+    "shortcuts": [
+        [
+            "qtpass.exe",
+            "QtPass"
+        ]
+    ],
+    "checkver": {
+        "github": "https://github.com/IJHack/QtPass/",
+        "regex": "tag/v([\\w.\\d]+)"
```

...

- `\d` is contained in `\w`
- Why would you use custom regex, when this release is covered by the native regex??

```suggestion
        "github": "https://github.com/IJHack/QtPass",
```

---

ahhhh I faint 😂